src/constants/earthEngineLayers/vegetation_original_MOD13Q1.js

```diff
@@ -4,11 +4,18 @@ import { BY_YEAR } from '../periods.js'
 
 export default {
     layer: EARTH_ENGINE_LAYER,
-    layerId: 'MODIS/061/MOD13Q1/VI',
+    layerId: 'MODIS/061/MOD13Q1/VI/16DAY',
```

Comment:
Will changing the layerId have any backwards compatibility issues with saved maps? Also, maybe this file should include "16 day" in the name, instead of "original"?